TransformationSystem/Client/TaskManager.py

```diff
@@ -138,6 +139,71 @@ def prepareTransformationTasks( self, transBody, taskDict, owner = '', ownerGrou
         return res
       ownerDN = res['Value'][0]
 
+    try:
+      transJson = json.loads(transBody)
+      self._multiOperationsBody( transJson, taskDict, ownerDN, ownerGroup )
+    except ValueError: ##json couldn't load
+      self._singleOperationsBody( transBody, taskDict, ownerDN, ownerGroup )
+
+    return S_OK( taskDict )
+
+  def _multiOperationsBody( self, transJson, taskDict, ownerDN, ownerGroup ):
+    """ deal with a Request that has multiple operations
+
+    :param transJson: list of lists of string and dictionaries, e.g.:
+
+      .. code :: python
+
+        body = [ ( "ReplicateAndRegister", { "SourceSE":"FOO-SRM", "TargetSE":"BAR-SRM" }),
+                 ( "RemoveReplica", { "TargetSE":"FOO-SRM" } ),
+               ]
+
+    :param dict taskDict: dictionary of tasks, modified in this function
+    :param str ownerDN: certificate DN used for the requests
+    :param str onwerGroup: dirac group used for the requests
+    :returns: None
+    """
+
+    for taskID in sorted( taskDict ):
+      paramDict = taskDict[taskID]
+      if not paramDict['InputData']:
```

Comment:
Yes, error message should be there.

Line 226 of this file also didn't check, so probably it is always there...
https://github.com/DIRACGrid/DIRAC/pull/3024/files/45c87a3e72fa618d0612c6af174277e9536f1730#diff-1118eb6fbeb8c43d317c646d5c45362aR226